FogCreek/Glitch-Community
# Loud roadrunner: hookify user editor

## Links
* Remix link: https://loud-roadrunner.glitch.me/

## Changes:
* convert UserEditor component into hook

## How To Test:
* log into https://loud-roadrunner.glitch.me/
* visit your user profile page
* verify that user actions (updaing profile, pinning projects) work like they do on prod
